app/controllers/browse_controller.rb

```diff
@@ -45,7 +45,7 @@ def self.modified_field
     #   The ordering of the field names is the order of the display
     config.add_facet_field solr_name("resource_type", :facetable), label: "Item Type", limit: 20
     config.add_facet_field solr_name("subject", :facetable), label: "Subject", limit: 20
-    config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
+    #config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
```

Comment:
Yeah, I agree. Kill dead code and trust your version control if you ever need it back. Commented out code is just a source of confusion.